ui/tests/integration/components/stat-text-test.js

```diff
@@ -0,0 +1,32 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+module('Integration | Component | StatText', function(hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders', async function(assert) {
+    // Set any properties with this.set('myProperty', 'value');
+    // Handle any actions with this.set('myAction', function(val) { ... });
+
+    await render(hbs`<StatText />`);
+
+    assert.equal(this.element.textContent.trim(), '');
```

Comment:
Maybe instead search for the name of the component?  It's a glimmer component, so you should see it rendered. Ping me if that doesn't make sense.